src/core/moves/MetropolisHastingsMove.cpp

```diff
@@ -300,11 +381,34 @@ void MetropolisHastingsMove::performMcmcMove( double prHeat, double lHeat, doubl
 
             if ( the_node->isClamped() )
             {
-                ln_likelihood_ratio += the_node->getLnProbabilityRatio();
+                try {
+                    ln_likelihood_ratio += the_node->getLnProbabilityRatio();
+                }
+                catch (RbException e)
```

Comment:
Catching exception by value.

Reply:
OK, I changed to catch by reference in commit a4f5cf43